### PR TITLE
MEN-5954: chore: Update Prepare a Raspberry Pi instructions

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md [hunk 1]
```diff
@@ -111,27 +122,31 @@ If you have a different network setup or encounter any issues, please see [the o
 
 
 
-### Option #2: Configure wireless network and enable SSH by attach a keyboard and monitor
+### Option #2: Configure your Raspberry Pi with attached a keyboard and monitor
```
Comment (oleorhagen) [inline 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:125]: S: `with a keyboard and monitor attached`

**Review by oleorhagen — APPROVED** [review]
Sweet

**Comment by lluiscampos** [comment]
Cherry-pick to:
* 3.4.x

**Comment by lluiscampos** [comment]
> Cherry-pick to:
> 
>     * 3.4.x

@mender-test-bot :point_up:

**Comment by lluiscampos** [comment]
@mender-test-bot cherry-pick to:
* 3.4.x

**Pull request merged**